google_functions_v2/sensor.py

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/python3
+
+from dataclasses import dataclass
+import typing as t
+import datetime as dt
+
+@dataclass
```

Comment:
Can you please add documentation describing purpose of the dataclass? Who generates it? Where does it come? What it represents?